projects/optic/src/commands/oas/verify.ts

```diff
@@ -38,8 +45,33 @@ export async function verifyCommand(): Promise<Command> {
     .option('--exit0', 'always exit 0')
     .option('--document <operations>', 'HTTP method and path pair(s) to add')
     .option('--patch', 'Patch existing operations to resolve diffs')
+    .option(
+      '--upload',
+      'Upload the verification data to optic cloud. Requires the spec to be in optic',
+      false
+    )
     .action(async (specPath) => {
       const analytics: { event: string; properties: any }[] = [];
+      const options = command.opts();
+
+      if (options.upload) {
+        if (options.document || options.patch) {
+          console.error(
+            'Cannot run upload with --document or --patch options.'
+          );
+          process.exitCode = 1;
+          return;
+        } else if (
+          config.vcs?.type !== VCS.Git ||
+          config.vcs.status === 'dirty'
```

Comment:
I'm not sure this is too strict, but it's not very useful in our current system to run verify --upload outside of a git repo / with a "dirty" working dir.

The spec wouldn't have any tags, and wouldn't really show up anywhere